docs/troubleshooting.md

```diff
@@ -3,51 +3,50 @@
 The Collector offers multiple ways to measure the health of the Collector
 as well as investigate issues.
 
+- The [zpages](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/zpagesextension/README.md)
+  extension, which by default is on port `55679`, can be used to check
+  receivers and exporters trace operations via `/debug/tracez`. `zpages`
+  often contains error logs that the collector does not emit.
 - By default, Prometheus metrics are exposed on port `8888` and path
-`/metrics`. A grafana dashboard can be found
-[here](https://grafana.com/grafana/dashboards/11575).
-
+  `/metrics`. A grafana dashboard can be found
+  [here](https://grafana.com/grafana/dashboards/11575).
 - Logs can be helpful in identifying issues. The verbosity level, which
-defaults to `INFO` can also be adjusted by passing the `--log-level` flag
-to the `otelcol` process. See `--help` for more details.
-
-- [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter#local-exporters)
-can be configured to inspect the data being processed by the Collector.
-
+  defaults to `INFO` can also be adjusted by passing the `--log-level` flag
+  to the `otelcol` process. See `--help` for more details.
+- [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter#general-information)
+  can be configured to inspect the data being processed by the Collector.
 - The [health_check](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/healthcheckextension/README.md)
-extension, which by default is on port `13133`, can be used to ensure
-the Collector is functioning properly.
-
-- The [zpages](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/zpagesextension/README.md)
-extension, which by default is on port `55679`, can be used to check
-receivers and exporters trace operations via `/debug/tracez`.
-
+  extension, which by default is on port `13133`, can be used to ensure
+  the Collector is functioning properly.
```

Comment:
Maybe "functioning properly" is slightly too strong wording? I think its just a liveness check right?